specificatie/publiek-rechtelijke-beperkingen.yaml

```diff
@@ -38,13 +38,16 @@ paths:
             example:
             - 196733.510
             - 439931.890
+        - $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/parameters/acceptCrs"
         - $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.1.0/api-specificatie/common.yaml#/components/parameters/contentCrs'
       responses:
         '200':
           description: "Zoekactie geslaagd"
           headers:
             api-version:
               $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/headers/api_version"
+            Content-Crs:
+              $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/headers/contentCrs"
```

Comment:
Hier wordt de v1.0.0 versie gebruikt van contentCrs en in regel 42 van de v1.1.0 versie. Is er een reden hiervoor? Anders is het handiger om het gelijk te houden

Reply:
ik denk dat er overal versie 1.1.0 moet worden gebruikt, ook op andere plekken. Ik zal dat in deze pull request doorvoeren.

Reply:
Is het niet mogelijk om het versienummer uit alle onderdelen weg te laten en alleen op te nemen in in de openapi.yaml ?

Reply:
> 
> 
> Is het niet mogelijk om het versienummer uit alle onderdelen weg te laten en alleen op te nemen in in de openapi.yaml ?

Ik neem aan dat je de version property bedoelt en niet de versie in de $ref-s. Het kan, maar dan voldoet zo'n oas bestand niet meer aan openapi specs als je een openapi validator gebruikt op het bestand

Reply:
Dank voor je antwoord Melvin, Ik had even niet goed naar de context van de discussie gekeken. Dat was inderdaad wat ik bedoelde.